### PR TITLE
allow to differentiate path_gdx_ref and path_gdx_refpolicycost

### PR DESCRIPTION
**Pull request by orichters**
For the scenario config, a new column name `path_gdx_refpolicycost` is introduced following the debate in https://github.com/remindmodel/development_issues/issues/10:

- `path_gdx_ref` is used as before for fixing until `cm_startyear`,
- `path_gdx_refpolicycost` is used as a baseline for policy cost calculation.

If no column `path_gdx_refpolicycost` is specified, `path_gdx_ref` is used as before, so this PR should not alter the working of any existing scenario config file.

`path_gdx_refpolicycost` is integrated in the determination of subsequent runs, so subsequent runs will wait until the run for comparison of policy cost is finished, although technically the `fulldata.gdx` is only required in the output generation at the end of the subsequent run. If this slows down the process, you can leave the `path_gdx_refpolicycost` cell empty and do the policy cost calculation manually after all runs are finished, or you can add a `_` after the run name in the cell to use an older, already finished run.

### CODE REVIEW

File: scripts/output/single/reportCEScalib.R [hunk 1]
```diff
@@ -17,8 +17,8 @@ require(colorspace)
 require(gdx)
 require(grid)
 
-gdx_name     <- "fulldata.gdx"        # name of the gdx  
-gdx_ref_name <- "input_ref.gdx"       # name of the reference gdx (for policy cost calculation)
+gdx_name     <- "fulldata.gdx"             # name of the gdx
+gdx_ref_name <- "input_refpolicycost.gdx"  # name of the reference gdx (for policy cost calculation)
```
Comment (LaviniaBaumstark) [inline scripts/output/single/reportCEScalib.R:21]: why is here this gdx loaded?

**Review by bs538 — COMMENTED** [review]
Thanks, I like the way this is set up. (Hope it's OK if I leave the technical review to Lavinia).

Is this supposed to be used in the default scenario config? If yes, you'd have to add a column with (SSPx/SDP/SDP_XX)-NPi as path_gdx_refpolicycost.

also putting @johanneskoch94 in the loop as he put together the previous (manual) policycost reporting via Rscript output.R

File: scripts/output/single/reportCEScalib.R (hunk 1 above)
Comment (orichters) [reply scripts/output/single/reportCEScalib.R:21]: In the sense of "why do we need it here anyway?" or "should be input_ref.gdx" as before?

Was introduced here: https://github.com/remindmodel/remind/commit/70d1ebd2d6a7b5d34d7abf06b279521744e5bdd2

**Comment by orichters** [comment]
@bs538: No, all the `path_gdx…` are not in the default config and cannot be, because you need to start with some run at some point before you can use it for some later comparison.

File: scripts/output/single/reportCEScalib.R (hunk 1 above)
Comment (LaviniaBaumstark) [reply scripts/output/single/reportCEScalib.R:21]: in the sense "should be input_ref.gdx" as before. I wondered why we calculate policy costs in this script. And actually, no input_ref.gdx file is used in the script could be deleted.

**Comment by bs538** [comment]
> @bs538: No, all the `path_gdx…` are not in the default config and cannot be, because you need to start with some run at some point before you can use it for some later comparison.

I thought in principle they can, just referring to runs that already happen previously in the cascade (like it's already done for path_gdx_ref, where the corresponding NDC run that has run previously is used). But using that here with NPi as path_gdx_policostref would mean that our cascade would get an additional step Base-NDC-*NPi*-(all other policy runs), which is probably undesirable. @LaviniaBaumstark using NDC (and not NPi) as ref for the fixing was decided on some point based on giving a better near-term picture right? (counter-intuitive as it is though...). perhaps this validation is a good moment to revisit this and check if that's still necessary to be done in this counter-intuitive order?

**Comment by orichters** [comment]
Yes, @bs538, you are right, this would imply a next step in the cascade and as I wrote above in the PR description, you have to decide whether you want to do that. Therefore, I did not change any scenario config files. This PR just offers the possibility to implement it this way or to select an older (already converged) run as refpolicycost run.

File: scripts/output/single/reportCEScalib.R (hunk 1 above)
Comment (orichters) [reply scripts/output/single/reportCEScalib.R]: done

**Review by LaviniaBaumstark — APPROVED** [review]

**Comment by LaviniaBaumstark** [comment]
I would not use this functionality in the current validation runs as it would slow down the process. We decided to use NDC runs for fixing the first time as those scenarios are in line with some long term climate policy assumptions. To ensure that NCD and NPi are identical during the fixing period we also run the NPi scenario afterwards based on the NDC scenario. I do not think that we should change this.

**Review by johanneskoch94 — APPROVED** [review]

**Pull request merged**